dev/io.openliberty.jakarta.rest.3.1.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -319,18 +446,19 @@
                     </systemPropertyVariables>
                     <testSourceDirectory>${basedir}${file.separarator}src${file.separarator}main${file.separarator}java${file.separarator}</testSourceDirectory>
                     <dependenciesToScan>
-                        <dependency>io.openliberty.jakarta.ws.rs:jakarta-restful-ws-tck</dependency>
-                    </dependenciesToScan>
+                         <dependency>io.openliberty.jakarta.ws.rs:jakarta-restful-ws-tck</dependency>   
+                     </dependenciesToScan>
                     <includes>
                         <!--  Example to run just one test. 
-                        <include>ee/jakarta/tck/ws/rs/spec/contextprovider/JsonbContextProviderIT</include> 
-                        -->                        
-                        <include>**/tck/**/*Test</include>
-                        <include>**/tck/**/*IT</include>
+                        <include>ee.jakarta.tck.ws.rs.signaturetest.jaxrs.JAXRSSigTestIT</include> -->                          
+                    
+                         <include>**/tck/**/*Test</include>
+                        <include>**/tck/**/*IT</include>  
```

Comment:
Do we need the includes at all?  Can we just do excludes and add the signature test to it?

Reply:
The includes are needed or the tests will not run for me so leaving that as-is.   And I have already switched to use the @Tag to exclude the se bootstrap group.